reports.go

```diff
@@ -46,15 +70,711 @@ type Report struct {
 	StateCriminalSearchIDs           []string   `json:"state_criminal_search_ids,omitempty"`
 	DocumentIDs                      []string   `json:"document_ids,omitempty"`
 	GeoIDs                           []string   `json:"geo_ids,omitempty"`
+	IdentityDocumentVerificationID   string     `json:"identity_document_verification_id"`
 }
 
 // Unmarshal ...
 func (r *Report) Unmarshal(b []byte) error {
 	return json.Unmarshal(b, &r)
 }
 
+const ssnTracePath = "/v1/ssn_traces"
+
+// GetSSNTrace return the ssn trace screening report
+func (r *Report) GetSSNTrace(ssnTraceID string, c *Client) (*SSNTrace, error) {
+	// since ID is empty we don't have anything to return. the caller should make sure if ssnTraceID is nil
+	if ssnTraceID == "" {
+		return nil, errEmptyID
+	}
+
+	rel, err := url.Parse(ssnTracePath)
+	if err != nil {
+		return nil, err
+	}
+
+	u := *c.BaseURL
+	u.Path = path.Join(u.Path, rel.String(), ssnTraceID)
+
+	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.SetBasicAuth(c.APIKey, "")
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, NewResponseError([]int{
+			resp.StatusCode,
```

Comment:
@chandradeepak the status here should be the `expected` status code, so it should be `http.StatusOK`.